AppCenterDistribute/AppCenterDistributeTests/MSDistributeConfigureTests.m

```diff
@@ -15,33 +15,32 @@ @interface MSDistributeConfigureTests : XCTestCase
 
 @implementation MSDistributeConfigureTests
 
-- (void)testConfigureFlagsBeforeStart {
+- (void)testDisableAutomaticCheckForUpdateBeforeStart {
 
   // If
-  [MSDistribute sharedInstance].distributeFlags = -1;
+  MSDistribute *distribute = [MSDistribute sharedInstance];
 
   // When
-  [MSDistribute configure:MSDistributeFlagsDisableAutomaticCheckForUpdate];
+  [MSDistribute disableAutomaticCheckForUpdates];
 
   // Then
-  XCTAssertEqual(MSDistributeFlagsDisableAutomaticCheckForUpdate, [MSDistribute sharedInstance].distributeFlags);
+  XCTAssertTrue(distribute.automaticCheckForUpdatesDisabled);
 }
 
-- (void)testConfigureFlagsDoesNotChangeAfterStart {
+- (void)testAutomaticCheckForUpdateDisabledDoesNotChangeAfterStart {
 
   // If
-  MSDistribute *distribute = [MSDistribute new];
-  distribute.distributeFlags = MSDistributeFlagsDisableAutomaticCheckForUpdate;
-
-  // When
+  MSDistribute *distribute = [MSDistribute sharedInstance];
```

Comment:
The test instantiated distribute every time in each test to avoid interfering each other as it is using a singleton instance.
If tests use `sharedInstance`, you should make sure instance is reset every test before starts. I would recommend to instantiate it so that it would isolate the test from others.